twinotter/plots/heights_and_legs.py

```diff
@@ -72,13 +74,21 @@ def generate(flight_data_path, legs_file):
         ax2.plot(ds_section.Time, ds_section.ALT_OXTS / 1000,
                  color=colors[label], linewidth=2, alpha=0.75)
 
+    if hasattr(ax2, 'secondary_yaxis'):
+        # `ax.secondary_yaxis` was added in matplotlib v3.1
+        ax2_fl = ax2.secondary_yaxis(
+            location=1.2,
+            functions=(lambda y: (y*1000*3.281)/100, lambda x: x)
```

Comment:
I think it will be useful to pull out that conversion factor at some point since we will probably want to use flight level on another plot. I've got a branch for deriving other variables from the dataset that I can add that change to so no need to do anything about it for now.

Reply:
Good idea. Yes we can also refactor this out to add this kind of flight level axes to other plots